packages/app-next/src/App.tsx

```diff
@@ -48,8 +55,26 @@ TODO:
 
 /* app.tsx */
 
+const fakePlugin = createPlugin({
+  id: 'fake',
+  extensions: [
+    createPageExtension({
+      id: 'catalog:entity',
+      defaultPath: '/fakeentity',
+      routeRef: entityRouteRef,
+      loader: async () => <div>hello 😅</div>,
```

Comment:
Is the intention to keep this around? 😅

Reply:
it's actually necessary as user settings plugin depends on `entityRouteRef`. I can polish a bit though 😅